doc/whats_new/v0.21.rst

```diff
@@ -12,6 +12,16 @@ Version 0.21.3
 Changelog
 ---------
 
+:mod:`sklearn.datasets`
+.......................
+
+- |Fix| :func:`fetch_california_housing`, :func:`fetch_covtype`,
```

Comment:
links are missing

Reply:
I think this may just work without the func. Otherwise you can use the ~ prefix

Reply:
they're resolved now.